lib/OAuth2.php

```diff
@@ -1001,8 +997,7 @@ protected function grantAccessTokenExtension(IOAuth2Client $client, array $input
         if (!($this->storage instanceof IOAuth2GrantExtension)) {
             throw new OAuth2ServerException(self::HTTP_BAD_REQUEST, self::ERROR_UNSUPPORTED_GRANT_TYPE);
         }
-        $uri = filter_var($inputData["grant_type"], FILTER_VALIDATE_URL);
```

Comment:
@Turneliusz actually, it doesn't work https://secure.php.net/manual/fr/filter.filters.validate.php#110411